packages/@react-spectrum/s2/src/Menu.tsx

```diff
@@ -81,22 +81,26 @@ export interface MenuProps<T> extends Omit<AriaMenuProps<T>, 'children' | 'style
   /**
    * The contents of the collection.
    */
-  children?: ReactNode | ((item: T) => ReactNode)
+  children?: ReactNode | ((item: T) => ReactNode),
+  /** Hides the default link out icons on menu items that open links in a new tab. */
+  hideLinkOutIcon?: boolean
```

Comment:
name?

Reply:
I think this name is fine actually haha

Reply:
Does it need the `icon` part of the name?